ThumbnailSequence.cpp

```diff
@@ -1560,7 +1560,7 @@ ThumbnailSequence::CompositeItem::CompositeItem(
 bool
 ThumbnailSequence::CompositeItem::incompleteThumbnail() const
 {
-	return dynamic_cast<IncompleteThumbnail*>(m_pThumb) != 0;
+	return m_pThumb == 0 && dynamic_cast<IncompleteThumbnail*>(m_pThumb) != 0;
```

Comment:
But you can't copy the unique_ptr and you can't move it because it's a member and you would delete it then, wether in constructor or the call to it doesn't matter!? I am new to c++11 but that is what I learn from the errors. 
And from what I understand the auto_ptr was replaced because one wanted stricter requirements. I think one has to use a shared_ptr or some other way to get around this problem. 

The other thing is I don't want to have to compile qt5 myself with special options  just because of one program.